pyproject.toml

```
@@ -156,15 +149,19 @@ Xu2019 = "pybamm.input.parameters.lithium_ion.Xu2019:get_parameter_values"
 ECM_Example = "pybamm.input.parameters.ecm.example_set:get_parameter_values"
 MSMR_Example = "pybamm.input.parameters.lithium_ion.MSMR_example_set:get_parameter_values"
 
-[tool.setuptools]
-include-package-data = true
+[tool.scikit-build]
+minimum-version = "0.9"
+build-dir = "build/{wheel_tag}"
+
+[tool.scikit-build.logging]
+level = "INFO"
+
+[tool.scikit-build.wheel]
+cmake = false
+expand-macos-universal-tags = true
```

```suggestion
cmake = false
```

We won't be compiling or publishing universal2 wheels for macOS anytime soon, so this setting is not needed.